code/modules/reagents/Chemistry-Reagents.dm

```diff
@@ -86,9 +89,29 @@
 /datum/reagent/proc/affect_touch(var/mob/living/carbon/M, var/alien, var/removed)
 	return
 
-/datum/reagent/proc/overdose(var/mob/living/carbon/M, var/alien) // Overdose effect. Doesn't happen instantly.
-	M.adjustToxLoss(REM)
-	return
+/datum/reagent/proc/overdose(var/mob/living/carbon/M, var/alien)
+//	M.overdosing -= REM
+	if(od_type == "heart")

```

Comment:
Yeah, you would be much better off with `organ/process_overdose()` or something like that instead of this.